core/src/main/java/com/novoda/noplayer/internal/mediaplayer/DelayedActionExecutor.java

```diff
@@ -0,0 +1,43 @@
+package com.novoda.noplayer.internal.mediaplayer;
+
+import android.os.Handler;
+
+import java.util.Iterator;
+import java.util.Map;
+
+class DelayedActionExecutor {
```

Comment:
Tracks all of the `Runnable`s for the workaround in `mediaPlayer` and removes all `Runnable`s when `reset` is called in `mediaPlayer`.